MathlibTest/SimplexCategory.lean

```diff
@@ -0,0 +1,73 @@
+import Mathlib.Tactic.SimplexCategory
+open SimplexCategory.Truncated
+
+variable (n : ℕ) (m : ℕ) (h : m ≤ n)
+
+#guard_expr ⦋2⦌₃ = (⟨SimplexCategory.mk 2, _⟩ : SimplexCategory.Truncated 3)
+#guard_expr ⦋1⦌ₙ₊₁ = (⟨SimplexCategory.mk 1, _⟩ : SimplexCategory.Truncated (n + 1))
+#guard_expr ⦋m⦌ₙ = (⟨SimplexCategory.mk m, _⟩ : SimplexCategory.Truncated n)
+
+#guard_expr ⦋2, by decide⦌₃ = (⟨SimplexCategory.mk 2, _⟩ : SimplexCategory.Truncated 3)
+#guard_expr ⦋1, by simp⦌ₙ₊₁ = (⟨SimplexCategory.mk 1, _⟩ : SimplexCategory.Truncated (n + 1))
+#guard_expr ⦋m, h⦌ₙ = (⟨SimplexCategory.mk m, _⟩ : SimplexCategory.Truncated n)
+
+section delaborator
+
+/-- info: ⦋2⦌₃ : CategoryTheory.FullSubcategory fun a => a.len ≤ 3 -/
+#guard_msgs in #check ⦋2⦌₃
+
+/-- info: ⦋m⦌ₙ : CategoryTheory.FullSubcategory fun a => a.len ≤ n -/
+#guard_msgs in #check ⦋m⦌ₙ
+
+/-- info: ⦋m⦌ₙ ₊ ₁ : CategoryTheory.FullSubcategory fun a => a.len ≤ n + 1 -/
+#guard_msgs in #check ⦋m⦌ₙ₊₁
```

Comment:
The test should match the input here:
```suggestion
/-- info: ⦋m⦌ₙ ₊ ₁ : CategoryTheory.FullSubcategory fun a => a.len ≤ n + 1 -/
#guard_msgs in #check ⦋m⦌ₙ ₊ ₁
```
If the spaces aren't in fact legal in the notation, then the delaborator should not emit them.